tests/test_query_execution.py

```
@@ -880,11 +880,18 @@ def test_output_formats(self, mock_echo, output_format, expected_ids, is_json_pa
-            # Text format uses console.print
+            # Text format uses console.print with a Rich Table
             env.ui.console.print.assert_called_once()
             output = env.ui.console.print.call_args[0][0]
+            # Rich Table: render to string for content assertions
+            from io import StringIO
+
+            from rich.console import Console as _RichConsole
+            buf = StringIO()
+            _RichConsole(file=buf, no_color=True, width=200).print(output)
+            rendered = buf.getvalue()
             for id in expected_ids:
```

Ruff (0.15.9)

[error] Variable `id` is shadowing a Python builtin

(A001)